sormas-backend/src/main/java/de/symeda/sormas/backend/region/RegionService.java

```diff
@@ -58,6 +58,24 @@ public RegionService() {
 		return em.createQuery(cq).getResultList();
 	}
 
+	public List<Region> getByExternalId(String externalId, boolean includeArchivedEntities) {
```

Comment:
Also, wouldn't it make sense to extract this method to `AbstractInfrastructureAdoService` ?

Reply:
All other getByName methods are in specific entity based services, maybe at a later point could be refactored to avoid these kind of similar methods

Reply:
Alright, let's keep it for later